web/src/components/navigation/navbarmobile.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import Logo from '../../img/tinysocial-logo.png';
+import MenuIcon from '@material-ui/icons/Menu';
+import {
+  BrowserRouter,
+  Route,
+} from 'react-router-dom';
+import {
+  AppBar,
+  Toolbar,
+  Grid,
+  IconButton,
+  Drawer,
+  List,
+  makeStyles,
+  Divider,
+  Button,
+  Typography, Tab, Tabs,
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  list: {
+    width: 250,
+  },
+  listItem: {
+    margin: 10,
+    marginLeft: 20,
+    color: '#000000',
```

Comment:
#000 does this work? 

Or can you check whether there's a predefined variable for this?

Reply:
#000 works. But changed it to 'black'.